description: fix for building with ozone_platform_wayland=false in GN_FLAGS in debian/rules

---

--- a/ui/ozone/BUILD.gn
+++ b/ui/ozone/BUILD.gn
@@ -19,9 +19,6 @@
 # Extra dependencies to pull into ui/ozone for built platforms.
 ozone_platform_deps = ozone_external_platform_deps
 
-# Extra dependencies to pull into ozone_unittests for built platforms.
-ozone_platform_test_deps = ozone_external_platform_test_deps
-
 # Extra dependencies to pull into ui_test_support for built platforms.
 ozone_platform_ui_test_support_deps =
     ozone_external_platform_ui_test_support_deps
